zh_cn.lang

```diff
@@ -18,30 +18,30 @@ gui.xaero_edit_mode=编辑模式
 gui.xaero_instructions=帮助
 gui.xaero_not_ingame=你必须处于游戏中才能使用编辑模式！
 gui.xaero_welcome_edit_mode=欢迎进入编辑模式！
-gui.xaero_howto_select=左键选择一个界面
-gui.xaero_howto_drag=左键拖动来移动界面
-gui.xaero_howto_deselect=右键来取消选择一个界面
-gui.xaero_howto_center=按下“C”来居中或取消居中一个界面
-gui.xaero_different_centered=部分界面有着不同的居中样式
-gui.xaero_howto_flip=按下“F”来翻转或取消翻转一个界面
-gui.xaero_howto_settings=按下“S”来快速访问所选界面的设置
-gui.xaero_howto_preset=点击“选择一个预设”按钮来选择一个预设
-gui.xaero_howto_save=点击“确认”按钮来保存你的更改
-gui.xaero_howto_cancel=点击“取消”按钮来取消你的更改
+gui.xaero_howto_select=左键选择一个界面。
+gui.xaero_howto_drag=左键拖动来移动界面。
+gui.xaero_howto_deselect=右键来取消选择一个界面。
+gui.xaero_howto_center=按下“C”来居中或取消居中一个界面。
+gui.xaero_different_centered=部分界面有着不同的居中样式。
+gui.xaero_howto_flip=按下“F”来翻转或取消翻转一个界面。
+gui.xaero_howto_settings=按下“S”来快速访问所选界面的设置。
+gui.xaero_howto_preset=点击“选择一个预设”按钮来选择一个预设。
+gui.xaero_howto_save=点击“确认”按钮来保存你的更改。
+gui.xaero_howto_cancel=点击“取消”按钮来取消你的更改。
 gui.xaero_minimap_settings=小地图设置
 gui.xaero_miscellaneous_settings=杂项设置
 gui.xaero_notification_settings=通知设置
 gui.xaero_better_pvp_settings=Better PvP 设置
-gui.xaero_server_disabled=服务器可能禁用了此 mod 的一些功能
+gui.xaero_server_disabled=服务器可能禁用了此 mod 的一些功能。
 gui.xaero_waypoints=路径点
-gui.xaero_box_potion_effects=一个显示当前激活的药水效果的界面
-gui.xaero_box_armour_status=一个显示当前装备和主手物品信息的界面
-gui.xaero_box_sprint=使用一个单独的按键（可配置的）来冲刺
-gui.xaero_box_sneak=允许你切换潜行状态，并且在潜行状态中不需要按住任何按键
-gui.xaero_box_notifications=为一些信息启用屏幕通知，如生命值低、饱食度低、氧气量低、苦力怕预警、或者骷髅怪预警
+gui.xaero_box_potion_effects=一个显示当前激活的药水效果的界面。
+gui.xaero_box_armour_status=一个显示当前装备和主手物品信息的界面。
+gui.xaero_box_sprint=使用一个单独的按键（可配置的）来冲刺。
+gui.xaero_box_sneak=允许你切换潜行状态，并且在潜行状态中不需要按住任何按键。
+gui.xaero_box_notifications=为一些信息启用屏幕通知，如生命值低、饱食度低、氧气量低、苦力怕预警、或者骷髅怪预警。
```

Comment:
苦力怕 means Creeper, and 骷髅怪 means Skeleton.

Reply:
I'm confused. Both seem incorrect. The notifications are not for creepers and skeletons exclusively. It's for explosions (TNT) and arrows shot by other entities.

Reply:
Is this notification shows before explosions/arrows shot or after?
If its after than it should be notification, otherwise it should be early-warning.

Reply:
The explosion notification is displayed while there is a primed TNT or a creeper near you. The arrow notification starts displaying when an arrow flies by very close to you.